src/components/BlogList/BlogList.jsx

```diff
@@ -9,14 +11,42 @@ const BlogList = ({ blogs, query }) => {
     );
   };
 
-  const filteredBlogs = searchedBlogs(blogs, query);
+  const filteredBlogsByTitle = searchedBlogs(blogs, query);
+
+  const filteredBlogsByTags = filteredBlogsByTitle?.filter(({ tags }) => {
+    if (filteredTags.length === 0) {
+      return true;
```

Comment:
```suggestion
      return;
```

Reply:
@ray-wtag bhai, Using return true we want to show all items when we don't select any tag, But if we use return without true would exclude everything, which isn't what we want.